sailthru/sailthru_client.py

```diff
@@ -333,10 +328,10 @@ def delete_template(self, template_name):
         data = {'template': template_name}
         return self.api_delete('template', data)
 
-    def save_template(self, template, template_fields=None):
-        template_fields = template_fields or {}
-        data = template_fields.copy()
-        data['template'] = template
+    def save_template(self, template, **template_fields):
```

Comment:
Future maintainers should change the client code to use `**kwargs` in the future